app/models/user.rb

```diff
@@ -267,30 +267,8 @@ def accepted_terms
     self.last_accepted_terms_at.to_i > User.terms_last_updated_at
   end
 
-  def self.terms_last_updated_at_by_page(page)
-    mapping = {
-      tos: 'tos_url',
-      tos_smooch: 'tos_smooch_url',
-      privacy_policy: 'privacy_policy_url'
-    }.with_indifferent_access
-    return 0 unless mapping.has_key?(page)
-    Rails.cache.fetch("last_updated_at:#{page}", expires_in: 24.hours) do
-      html = URI(CheckConfig.get(mapping[page], nil, :json)).open(read_timeout: 5, open_timeout: 5).read
-      date = ActionController::Base.helpers.strip_tags(html).match(/Last modified(?<modified_date>(Jan(?:uary)?|Feb(?:ruary)?|Mar(?:ch)?|Apr(?:il)?|May|Jun(?:e)?|Jul(?:y)?|Aug(?:ust)?|Sep(?:tember)?|Oct(?:ober)?|Nov(?:ember)?|Dec(?:ember)?)\s\d{1,2},\s\d{2,4})/)["modified_date"]
-      Time.parse(date).to_i
-    end
-  end
-
   def self.terms_last_updated_at
-    begin
-      tos = User.terms_last_updated_at_by_page(:tos)
-      pp = User.terms_last_updated_at_by_page(:privacy_policy)
-      tos > pp ? tos : pp
-    rescue StandardError => e
-      error = ToSOrPrivacyPolicyReadError.new(e)
-      CheckSentry.notify(error)
-      0
-    end
+    Rails.cache.read('terms_last_updated_at') || 0
```

Comment:
Please add a comment saying that we expect this value to be an integer timestamp.

Reply:
Added a comment and for a double check I added `.to_i`